lib/msal-common/src/config/ClientConfiguration.ts

```diff
@@ -91,7 +91,6 @@ export type AuthOptions = {
 export type SystemOptions = {
     tokenRenewalOffsetSeconds?: number;
     preventCorsPreflight?: boolean;
-    proxyUrl?: string;
```

Comment:
This is a great cleanup, but is this ^ ^ not a breaking change? How does the user configure this setting now?

Reply:
It's still passed in via the msal's configuration's system options - just like the HttpAgent

Reply:
It will be a breaking change for `msal-common`, we need to release a major version for it.

Reply:
+1 please make sure you get versions updated with this PR.